tests/drivers/at/main.c

```diff
@@ -51,9 +53,15 @@ static int init(int argc, char **argv)
         printf("Wrong UART device number - should be in range 0-%d.\n", UART_NUMOF - 1);
         return 1;
     }
-
-    int res = at_dev_init(&at_dev, UART_DEV(uart), baudrate, buf, sizeof(buf));
-
+    at_dev_init_t at_init_struct = {
```

Comment:
I am voting to drop the `_struct` suffix from the variable name. It's too much [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation) for my taste. Isn't `at_init_params` a better name?